.eslintignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+coverage
+static_compiled
+venv
+jinja2
+core
+workflows
```

Comment:
Nit: Can these paths be expanded (eg `/ons_alpha/core`), just in case the directory names are reused elsewhere

Reply:
sure - I'll leave ones like `node_modules` which should never be linted even if there was more than one directory.